src/main/java/com/google/api/codegen/nodejs/JSDocCommentFixer.java

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.nodejs;
+
+import com.google.api.codegen.CommentPatterns;
+import com.google.common.base.Splitter;
+
+import java.util.regex.Matcher;
+
+/**
+ * Utility class for formatting source comments to follow JSDoc style.
+ */
+public class JSDocCommentFixer {
+
+  /**
+   * Returns a JSDoc-formatted comment string.
+   */
+  public static String jsdocify(String comment) {
+    comment = jsdocifyProtoMarkdownLinks(comment);
+    comment = jsdocifyCloudMarkdownLinks(comment);
+    return cleanupTrailingWhitespaces(comment);
+  }
+
+  private static String protoToRubyDoc(String comment) {
```

Comment:
Actually, I don't see where this method is being used -- if it's not, remove it?